Docs 36 labels multiseries (#12772)
metabase/metabase: docs/users-guide/09-multi-series-charting.md

- New users per day vs. returning users per day.
- Orders per day from a few different product lines.

> ### Two ways to display data side-by-side

Reviewer: ```suggestion
### Displaying data side-by-side
```

Reviewer: There's actually a third thing you can do, which is adding a second metric to your question. E.g., you can do a Sum of Subtotal and a Count of Rows by Created At on the Orders table.

You can also do a join in Custom Question mode, where you join e.g. count of rows by created at with a different time series saved question, like Users Over Time.

> 1. [**Use the notebook editor to ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) (or in SQL, if you’re fancy). Example: the count of users by region over time.

Reviewer: I think above this you could add:

"In Metabase, there are two main ways to get data side-by-side:"

> 1. [**Use the notebook editor to ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) (or in SQL, if you’re fancy). Example: the count of users by region over time.

Reviewer: You can add multiple dimensions in both Simple and Custom mode. In Simple mode, you do it by clicking the + next to a column in the Group By part of the Summarize sidebar.

```suggestion
1. [**Ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) with the Simple or Custom query builders (or in SQL, if you’re fancy). Example: the count of users by region over time.
```

> 2. [**Combine two saved questions**](#combining-two-saved-questions) that share a common dimension (like time) on a dashboard. For example, let me see revenue over time and cost over time together.

Reviewer: ```suggestion
2. [**Combine two saved questions**](#combining-two-saved-questions) that share a common dimension (like time) on a dashboard. For example, you could look at revenue over time and costs over time together.
```

> ### Using the notebook editor to create a multi-series visualization

Reviewer: Ah, so I'm seeing here that there's some terminology confusion: "the notebook editor" is what we sometimes call the Custom Question mode. What you're describing below is the Simple Question mode.

The way you'd do this same thing in the Custom Question mode is to add two columns in the group-by section of the Summarize block.

> If you’re creating a new question, you can view the results as a multi-series visualization by summarizing your data and grouping it into two or more groups.

Reviewer: I think it has to be _exactly_ two groups. If you try breaking out the count of orders by Created_At, Product.Category, _and_ People.State, you'll have to pick which two of the breakouts to use for the x-axis and for the series breakout; the third breakout won't be used.

As an example, we might want to see which website or service is referring the most people to our website. In the sample dataset that ships with Metabase this would involve using the `Source` and `Created At` columns of the **People** table.

To create the multi-series chart, we'd select the **People**, click on the **Summarize** button in the upper right, then add `Source` and `Created At` as groupings (the `count of rows` metric that we want is already selected by default). Be sure to click the plus button to the right of your selection, so Metabase knows to add the grouping. Otherwise, Metabase will switch to that grouping. [Learn more about asking questions](04-asking-questions.md).

Metabase will automatically display a multi-series line chart visualization of how each referrer has performed for us.

![multi-series in the query builder](images/multi-series-charts/multi-series_query_builder.png)

> > You won’t be able to add another saved question to multi-series visualizations made in this fashion. Metabase can only visualize up to 20 values of a dimension at once, so if you're selecting a field that contains a lot of values, you may need to filter the values.

Reviewer: ```suggestion
> You won’t be able to add another saved question to multi-series visualizations made in this fashion. Metabase can visualize up to 100 distinct values of a dimension at once, so if you're selecting a field that contains a lot of values, you might need to filter the values.
```

### Combining Number charts

If you need to compare single numbers to get a sense of how they differ, Metabase can turn multiple number charts into a bar chart. 

As above, while editing a dashboard, hover over a number chart of your choice, and click on the icon with the **+** and bar chart icon to add a saved question. 

![Add bar chart](images/multi-series-charts/add_bar_chart.png)

Use the search bar to find other saved question(s) that you’d like to see represented on the bar chart, and click the checkbox to add them to your chart. In this case, we added **Widget orders** to compare them to **Gadget orders**.

![From numbers to bar chart](images/multi-series-charts/numbers_to_bar_chart.png)

### Multi-series charts, values, and legibility

When displaying multiple series, it’s important to keep legibility in mind. Combining many series can sometimes decrease the communication value of the data.

> Metabase allows you to add values to multi-series charts, but go easy on this feature, especially on charts with lots of data points. Adding values to multiple series, each with many data points, can make charts more difficult to read.

Reviewer: Worth noting here is that you can turn values on for just _one_ series (or a few series), by expanding that series' details in the Display tab, and turning on the display values toggle for it. You can also toggle on values for the whole chart, then turn it off selectively for individual series.

> From the **Visualization -> Display** options, you can toggle a slider to **Show values on data points**. Metabase will do its best to fit as many values as can fit nicely. You can also force Metabase to (begrudgingly) show values for all data points, by setting the **Values to show** to **All**. 

Reviewer: Nitpicky, but I think "sliders" are technically those controls that allow you to pick a numeric value along a range of values, or to pick e.g. a volume level. On/off buttons are "toggles." So I'd say, "you can toggle on 'Show values on data points,'" or "you can turn on the 'Show values on data points' toggle."
